Sources/MapboxNavigation/RoutesPresentationStyle.swift

```diff
@@ -1,4 +1,6 @@
 import Foundation
+import MapboxMaps
+import MapboxDirections
```

Comment:
Do we really need `MapboxDirections` here?

Reply:
No, I'll remove it.